birdhouse/README.rst

```diff
@@ -350,6 +350,21 @@ not able to access protected URLs:
   not be fully functional when self-signed certificates are enabled. For example, accessing other components through
   the JupyterLab interface may fail with an ``SSLError``.
 
+Docker rootless configuration
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you are using `Docker Rootless Mode <https://docs.docker.com/engine/security/rootless/>`_ on your machine,
+you might need to execute the following command to allow the `proxy` (Nginx) service to connect to the relevant
+HTTP ports.
+
+.. code-block:: shell
+    sudo sysctl -w net.ipv4.ip_unprivileged_port_start=80
+
+This could be required when encountering errors such as the following when invoking ``birdhouse compose up -d``.
```

Comment:
I usually use `bin/birdhouse` in the documentation so that it's clear what we're talking about (in case the user hasn't added the interface script to their PATH)

Reply:
Good point. Will adjust.